app/src/pages/api/report/list/index.ts

```diff
@@ -0,0 +1,94 @@
+import {
+    HttpMethod,
+    IApiErrorResponse,
+    IReport,
+    INTERNAL_SERVER_ERROR,
+    METHOD_NOT_ALLOWED,
+} from '@/models';
+import { ReportDbClient } from '@/utils/clients/ReportDbClient';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+/** Return a list of report and pagination token upon a LIST request */
+export interface IListReportResponse {
+    /** Reports */
+    reports: IReport[];
+    /** Pagination token */
+    paginationToken: string | undefined;
+}
+
+/** NOTE: function to mock DB lisitng because I don't have access to create report page or permission to create new item in AWS
+    TODO: will need to switch to real DB function eventually
+*/
+const mockListReportsFromDb = (): { reports: IReport[]; paginationToken: string | undefined } => {
+    return {
+        reports: [
+            {
+                reportId: 'report1',
+                name: 'John Doe',
+                emailAddress: 'john.doe@example.com',
+                phoneNumber: '123-456-7890',
+                reportCategory: ['Illegal Dumping'],
+                address: '123 Main St',
```

Comment:
@andreybutenko I made this mock function. Is it a good idea to put it here or should I put it somewhere else like a test or util folder? What do you think?

Reply:
Good question! I think if we were using this data in multiple places or for unit tests, it should be in a test constants folder! But since this will be replaced shortly anyway, it is fine here until it is replaced with real db interaction :)